.env.mainnet

```
@@ -1,5 +1,4 @@
 export REACT_APP_IS_MAINNET=true
 export REACT_APP_CONTRACT_ADDRESS=0x00000000219ab540356cBB839Cbe05303d7705Fa
 export REACT_APP_ETH_REQUIREMENT=524288
```

The .env files are for developers to switch between the network easily. You can type `source .env.pyrmont` or `source .env.mainnet` locally. I think it would be safer to add `REACT_APP_TESTNET_LAUNCHPAD_NAME`, so the future short-term or long-term testnet (*not* Pyrmont) envs won't overwrite it.

```suggestion
export REACT_APP_ETH_REQUIREMENT=524288
export REACT_APP_TESTNET_LAUNCHPAD_NAME=Pyrmont
```